src/data/tutorials.json

```diff
@@ -1,4 +1,88 @@
 [
+  {
+    "url": "https://www.youtube.com/watch?v=M576WGiDBdQ",
+    "title": "Solidity, Blockchain, and Smart Contract Course – Beginner to Expert Python Tutorial",
+    "description": "This course will give you a full introduction into all of the core concepts in blockchain, smart contracts, solidity, NFTs/ERC721s, ERC20s, Coding Decentralized Finance (DeFi), python and solidity, Chainlink, Ethereum, upgradable smart contracts, and full stack blockchain development. ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "brownie", "web3.py", "smart-contracts", "react", "blockchain", "video", "IPFS", "oracles", "NFT", "ERC20", "upgrades", "python", "mocking", "testing", "Aave", "chainlink", "ganache"],
+    "skillLevel": "beginner",
+    "timeToRead": "960",
+    "lang": "en",
+    "publishDate": "09/09/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=9oERTH9Bkw0",
+    "title": "How to make NFT Art with On-Chain Metadata | FULL HARDHART / JS TUTORIAL!",
+    "description": "We explore the world of using SVGs to generate random NFT ImageURIs and Metadata 100% on-chain.  ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "ethers.js", "hardhat", "NFT", "smart-contracts", "javascript", "chainlink", "video"],
+    "skillLevel": "beginner",
+    "timeToRead": "180",
+    "lang": "en",
+    "publishDate": "09/03/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=TmNGAvI-RUA",
+    "title": "Leveraged Trading in DeFi | Python, Brownie, Aave, Uni/SushiSwap",
+    "description": "Leveraged trading is a common strategy to use in traditional finance, and they are even easier to do in DeFi. ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart-contracts", "python", "video"],
+    "skillLevel": "beginner",
+    "timeToRead": "34",
+    "lang": "en",
+    "publishDate": "06/29/2021"
+  },
+  {
+    "url": "https://www.youtube.com/watch?v=bdXJmWajZRY",
+    "title": "Upgrading your Smart Contracts | A Tutorial & Introduction",
+    "description": "Learn how to make contracts that use flash loans. Using Brownie, Solidity, Aave. ",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart-contracts", "python", "video", "defi"],
+    "skillLevel": "intermediate",
+    "timeToRead": "17",
+    "lang": "en",
+    "publishDate": "04/25/2021"
+  },
```

Comment:
```suggestion
  {
    "url": "https://www.youtube.com/watch?v=bdXJmWajZRY",
    "title": "Using flash loans in your smart contracts",
    "description": "Learn how to make contracts that use flash loans. Using Brownie, Solidity, Aave. ",
    "author": "Patrick Collins",
    "authorGithub": "https://github.com/PatrickAlphaC",
    "tags": ["solidity", "web3.py", "brownie", "Aave", "smart-contracts", "python", "video", "defi"],
    "skillLevel": "intermediate",
    "timeToRead": "17",
    "lang": "en",
    "publishDate": "04/25/2021"
  },
```

Reply:
I think this one is correct no? Does that link give you a flash loan video? @minimalsm